src/elements/content-sidebar/activity-feed/task-form/TaskForm.js

```diff
@@ -331,13 +329,13 @@ class TaskForm extends React.Component<Props, State> {
         const { dueDate, approvers, message, formValidityState, isLoading, completionRule } = this.state;
         const inputContainerClassNames = classNames('bcs-task-input-container', 'bcs-task-input-is-open', className);
         const isCreateEditMode = editMode === TASK_EDIT_MODE_CREATE;
-        const renderApprovers = convertAssigneesToSelectorItems(approvers);
+        const selectedApprovers = convertAssigneesToSelectorItems(approvers);
 
         // filter out selected approvers
         // map to datalist item format
         const approverOptions = approverSelectorContacts
-            .filter(({ id }) => !renderApprovers.find(({ value }) => value === id))
-            .map(({ id, item }) => ({ ...item, text: item.name, value: id }));
+            .filter(({ id }) => !selectedApprovers.find(({ value }) => value === id))
+            .map(({ id, item = {} }) => ({ ...item, text: item.name, value: id, item }));
```

Comment:
Should we default `item` to an empty object at a higher level to avoid needing this in two places?

Reply:
Checked and we can remove this .map, the data is formatted enough already.